more supervisor_id columns

snopoke opened the pull request:
The UCR was added in Jan so was missed before. The other tables were not marked for distribution in the sheet but should be.

@sravfeyn these will need backfilling as well

snopoke committed:
add supervisor_id to cbe_form

snopoke committed:
add supervisor_id to ccs_record_monthly and child_health_monthly

snopoke commented on custom/icds_reports/migrations/0107_child_cases_ccs_record_supervisor_id.py:
```diff
@@ -0,0 +1,18 @@
+
+    operations = [
+        migrations.RunSQL("ALTER TABLE ccs_record_monthly ADD COLUMN supervisor_id text"),
+        migrations.RunSQL("ALTER TABLE child_health_monthly ADD COLUMN supervisor_id text")
```
these have child tables for each month

emord replied:
Was this intentional for this PR?

kaapstorm reviewed (approved):
I don't have much context, but this looks fine.

emord commented:
Actually, I i think that we should add the superviosr_id to the aggregation script so that these are filled in for current months

snopoke committed:
add supervisor ID to agg queries

snopoke commented:
> Actually, I i think that we should add the superviosr_id to the aggregation script so that these are filled in for current months

added here 40dc0f5cd72 - I still need to fix tests though. Will look at that tomorrow

emord committed:
Add supervisor_id to ccs_record_monthly and child_health_monthly models

emord committed:
Add supervisor_id to test ouptut.

Generated using query such as:

```\copy (SELECT * FROM "child_health_monthly_2017-05-01" ORDER BY
awc_id, case_id) TO
'/home/jemord/workspace/commcare-hq/custom/icds_reports/tests/outputs/child_health_monthly_2017-05-01_sorted.csv'
CSV HEADER;
```

Verified with:

`git diff --color-words="[^[:space:],]+" -- custom/icds_reports/tests/outputs/child_health_monthly_2017-04-01_sorted.csv`

emord commented:
@snopoke went ahead and fixed tests with the commands i used in the commit message

snopoke commented:
That's a nice trick with `--color-words`

emord commented on custom/icds_reports/ucr/data_sources/cbe_form.json:
```diff
@@ -55,6 +55,24 @@
+        "transform": {},
+        "is_nullable": true,
+        "type": "expression",
+        "column_id": "supervisor_id"
```
There are currently 45 forms in this datasource. I'll just rebulid the whole table instead of writing a query
